client/jetpack-cloud/sections/partner-portal/add-stored-credit-card/index.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { useDispatch } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import CardHeading from 'calypso/components/card-heading';
+import Gridicon from 'calypso/components/gridicon';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function AddStoredCreditCard(): ReactElement {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const navigateToCreateMethod = () => {
+		dispatch(
+			recordTracksEvent( 'calypso_partner_portal_license_list_empty_issue_license_click' )
+		);
+	};
+
+	return (
+		<a
+			className="add-stored-credit-card"
+			href="/partner-portal/payment-method/add"
+			onClick={ navigateToCreateMethod }
+		>
+			<div className="add-stored-credit-card__content">
+				<CardHeading className="add-stored-credit-card__title" tagName="h3">
+					<Gridicon key="add-card-icon" icon="add-outline" size={ 24 } />
+					<span key="add-card-text">{ translate( 'New credit card' ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'New Credit Card' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-partner-portal-payment-method-list)